help/domains/bio/bioinformatics.md

```diff
@@ -0,0 +1,406 @@
+
+### Multiple Sequence Alignment (MSA)
+
+The [Multiple Sequence Alignment](https://en.wikipedia.org/wiki/Multiple_sequence_alignment)
+(MSA) feature allows you to create alignments of biological sequences.
+This action generates a new column containing the aligned sequences.
+MSA works for macromolecules in both FASTA and HELM formats.
+
+If your data has already been clustered, Datagrok provides an option to specify a "cluster" column
+to align sequences only within the same cluster.
+
+![Multiple Sequence Alignment dialog](img/MSA_dialog.png)
+
+For DNA, RNA, and natural peptides, Datagrok utilizes
+[KAlign](https://github.com/TimoLassmann/kalign) to perform the alignment.
+It is an open-source tool under GNU GPL, so it can be modified
+to work with custom substitution matrices for
+sequences.
+
+When dealing with peptides containing non-natural amino acids,
+Datagrok employs [PepSeA](https://github.com/Merck/PepSeA) for alignment purposes.
+It allows for alignment of multiple linear peptide sequences in HELM notation, with lengths up
+to 256 non-natural amino acids. _PepSeA_ uses a substitution matrix calculated with Rapid Overlay of Chemical Structures
+Similarities Across ChEMBL 28 HELM Monomers.
+
+### Compositional analysis
+
+Allows you to generate a "weblogo" representation, also known as a
+[sequence logo](https://en.wikipedia.org/wiki/Sequence_logo).
+This visual representation illustrates the letter composition for each position in a sequence.
+
+Typically, a sequence logo is created from a set of aligned sequences
+and helps to identify patterns and variations within the sequences.
+For example, it is commonly used to visualize protein-binding sites in DNA or functional motives in proteins.
+
+![WebLogo representation of sequence composition](img/WebLogo.png)
+
+WebLogo dynamically reflects the sequence sets filtering and allows the
+user to select a subsequence by choosing the residue at a specified position. The tooltip displays the number of
+sequences with a specific monomer at a particular position
+
+![Logo plot](../../uploads/macromolecules/macromolecules-10.gif "Logo plot")
+
+<!---
+
+Have not been tested by me
+
+Another way to perform the composition analysis for MSA data is to use composition Bar charts as any plot in the
+Datagrok reflects all user actions as filtering and selection.
+
+![Bar chart plot](../../uploads/macromolecules/macromolecules-bar-chart-800.gif "Bar chart")
+
+-->
+
+### Similarity search
+
+A linked dataframe displaying the most similar sequences to a selected sequence from the initial table.
+You have the flexibility to adjust the number of sequences displayed and set a similarity cutoff
+in the dataframe properties.
+
+### Diversity search
+
+Creates a new dataframe containing the most diverse sequences within a given dataframe.
+Unlike the [similarity search] feature, this dataframe is not interactive
+and does not respond to sequence selections in the initial dataframe.
+
```

Comment:
Sim and Div search - put them to the single part - Similarity and Diversity search
"Unlike the [similarity search] feature, this dataframe is not interactive
and does not respond to sequence selections in the initial dataframe." - remove this

Reply:
Fixed